Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.10-slim-bullseye AS compile-image
 WORKDIR /app
```

Comment:
Add Unit Test for all methods used or changed in the existing flow ensuring maximum code coverage